src/cluster/src/install.rs

```diff
@@ -234,7 +251,7 @@ impl ClusterInstallerBuilder {
     /// # Example
     ///
     /// ```no_run
-    /// use fluvio_cluster::ClusterInstaller;
+    /// # use fluvio_cluster::ClusterInstaller;
     /// let installer = ClusterInstaller::new()
```

Comment:
shouldn't this be refer to ClusterInstallBuilder?

Reply:
The way this is set up is that the user doesn't ever need to interact with the `ClusterInstallerBuilder` type. Currently, the `ClusterInstaller::new()` function actually returns a `ClusterInstallerBuilder`, which is why you see this in the documentation. I introduced #339 in order to make things a little bit nicer by making the function names more accurately represent what is happening. If we merge #339, this line would be rewritten as `let installer = ClusterInstaller::builder()...`